### PR TITLE
Add filter to Operator API reference generation

### PR DESCRIPTION
**Pull request by tmjd**
## Description

Add a config file for Operator API reference generation.
I've also included the reference built with operator master.
I also updated the reference links to K8s v1.18 from v1.15.

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: reference/installation/config.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
```
Comment (ozdanborne) [inline reference/installation/config.json:1]: this looks like a good config file for OS, but should we maintain a second config file for enterprise?

File: reference/installation/config.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    "hideMemberFields": [
+        "TypeMeta"
+    ],
+    "hideTypePatterns": [
+        "ParseError$",
+        "List$",
+        "AmazonCloudIntegration",
```
Comment (ozdanborne) [inline reference/installation/config.json:8]: When I thought about this problem earlier this week, I was disappointed that we'd need to maintain a list of all enterprise CRDs here. 

I wonder if instead, we can place a kube-builder annotation on all of our OS and/or enterprise CRDs to indicate which they are, then just filter based on that here.

Comment (tmjd) [reply reference/installation/config.json:8]: That's what I tried to do on my 1st attempt (a while back) and didn't figure it out. This seemed like the easy route to keep the Enterprise resources out, at least until we can figure out the more involved/complicated route of a kube-builder annotation.

File: reference/installation/config.json (hunk 1 above)
Comment (tmjd) [reply reference/installation/config.json:1]: Yeah but these are not the enterprise docs so there is no need to have that here.

Comment (ozdanborne) [reply reference/installation/config.json:1]: oh duh

**Review by ozdanborne — APPROVED** [review]

**Pull request merged**